tests/data/format/multi_style/numpydoc_parameter_spacing.py.out

```diff
@@ -0,0 +1,74 @@
+"""Example module for numpydoc docstring style.
+
+References
+----------
+NumPy docstring style guide:
+https://numpydoc.readthedocs.io/en/latest/format.html#documenting-modules
+"""
+import math
+
+EULER_NUMBER = math.e
+"""Euler's number.
+
+Not related to Euler's constant (sometimes called the Euler-Mascheroni
+constant.
+
+Notes
+-----
+It is the limit of ``(1 + 1/n)**n`` as n approaches infinity, so it
+is used in the equation for continuously-compouned interest.
+
+It is also the sum of the reciprocals of the whole numbers starting
+with zero, which is related to some calculus-related properties
+mathemeticians find elegant.
+
+References
+----------
+E (mathematical constant)
+https://en.wikipedia.org/wiki/E_(mathematical_constant)
+"""
+
+
+def sincos(theta):
+    """Returns.
```

Comment:
Oh this is becoming more difficult than I hoped. It seems both styles are somewhat incompatible with each other. This is the missing section name, which is now being used as a summary/title...

I'll have to think about this. Perhaps we should fix this in another PR, but I like to check if this is easily fixable.

Reply:
Well ... if you believe that the issue is that the styles are mutually exclusive in some instances, enforcing that is not hard internally.

Reply:
I think the issue that they are currently mutually exclusive. Perhaps we can fix this by changing the order in which formatters are called. I hope to have time somewhere this week to look into this.